templates/user_rank.html

```diff
@@ -0,0 +1,126 @@
+{% load staticfiles %}
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+        <title>Wikimedia User Ranking Tool</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="https://tools-static.wmflabs.org/cdnjs/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css">
+        <link rel="stylesheet" href="https://tools-static.wmflabs.org/cdnjs/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <script src="https://tools-static.wmflabs.org/cdnjs/ajax/libs/jquery/3.2.1/jquery.js"></script>
+        <script src="https://tools-static.wmflabs.org/cdnjs/ajax/libs/twitter-bootstrap/4.0.0-beta/js/bootstrap.min.js"></script>
+        <link rel="stylesheet" type="text/css" href={% static "css/main.css" %}>
+        <script type="text/javascript" src={% static "js/main.js" %}></script>
+</head>
+
+<body>
+
+        <div class="container">
+                <div class="row">
+                        <div class="col-md-6">
+                                <img src={% static "img/user-rank-tool.png" %} align="left">
+                        </div>
+                        <div class="col-md-6 search-box-container main-heading">
+                                <h2>Whose rank would you like to know?</h2>
+                                <div id="wrap">
+                                        <form autocomplete="on" method="post">
+                                                {% csrf_token %}
+                                                {% if isPost %}
+                                                <input id="username" value={{ userData.username }} name="username" type="text" placeholder="Enter his/her username...">
+                                                {% else %}
+                                                <input id="username" name="username" type="text" placeholder="Enter his/her username...">
```

Comment:
It's better to avoid [gender binary](https://en.wikipedia.org/wiki/Gender_binary) phrasing in UI messages. This one sounds a bit awkward in English anyway, normally one would just write "Enter username..."

Reply:
Done.